docs/user/dql/functions.rst

```diff
@@ -2258,8 +2262,9 @@ Description
 >>>>>>>>>>>
 
 The week_of_year function is a synonym for the `week`_ function.
+If an argument of type `TIME` is given, the function will use the current date to calculate an output.
```

Comment:
```suggestion
If an argument of type `TIME` is given, the function will use the current date.
```

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/199/commits/eac4b4e2ca1fb8a3041a572b9915a9be212981e4